### DISCUSSION TITLE
Cura Start Gcode

### DISCUSSION THREAD

**Discussion by Patpend1973** [mriscoc/Ender3V2S1#1065] [Q&A]
I have been working on getting this firmware working right. When I change Cura's starting gcode to what https://github.com/mriscoc/Ender3V2S1/wiki/Slicer-G-code-Scripts has, I slice then import to octoprint. When I start the print I get an error, 
Slicer Settings Missing
Some slicer settings were missing from your gcode file. Missing Settings: Retraction Length,Retraction Speed,Deretraction Speed,Z Lift Height,X/Y Travel Speed,Z Travel Speed

What have I screwed up? Thanks in advance.

**Comment by mriscoc** [comment]
Check your Octoprint plugins.

**Reply by Patpend1973** [reply]
Bump

**Comment by Patpend1973** [comment]
I wished that helped but it really doesn't. What am I checking for?

**Comment by Patpend1973** [comment]
Still looking for some help here. What octo print settings am I checking? I used your github pages octoprint setting suggestions. Also the video that you sent me talks about g29 and p1. I have neither as I do not run a mesh before every print
